samples/spring-boot/pom.xml

```diff
@@ -178,6 +179,39 @@
                                     </fmp-service>
                                 </config>
                             </enricher>
+                            <!-- sample assembly configuration. make sure all the directories configured exist. -->
+                            <!--<images>
+                                <image>
```

Comment:
Is it okay to leave this code as commented?

Reply:
I have knowingly added this here in the sample pom. We actually don't have any sample demonstrating AssemblyConfig feature so I thought this will be helpful.